libs/features/personalization/personalization.js

```diff
@@ -7,7 +7,7 @@ import {
 import { getEntitlementMap } from './entitlements.js';
 
 /* c8 ignore start */
-const PHONE_SIZE = window.screen.width < 768 || window.screen.height < 768;
+const PHONE_SIZE = window.screen.width < 600 || window.screen.height < 600;
```

Comment:
Using `600` as breakpoint would exclude some smartphones when on landscape mode. Typically the smallest tablet has `768` of width (iPad mini), why not use `767` instead?

Reply:
That's why the line checks width and height. Either being smaller avoids miscategorizing a device in landscape.  Essentially, I trying to check that the SMALLEST dimension is less than our checkpoint.

Someone pointed me to a site that lists the viewports for a bunch of devices and I've updated to go down to 550 instead.  Trying to catch the smaller tablets.

Reply:
Sorry, I should mention.  This came up in reference to a specific use case: Samsung Galaxy tablets.  They are smaller than iPads.